README.md

```diff
@@ -24,10 +24,10 @@ Detailed documentation can be found on our [GitHub Documentation Page](http://vm
 
 The download consists of 2 parts:
 
-1. VIB (vDVS driver): The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
-2. Managed plugin (vDVS plugin): Plugin is available on [Docker store](https://store.docker.com/plugins/e15dc9d5-e20e-4fb8-8876-9615e6e6e852?tab=description).
+1. VIB (vSphere Storage driver for Docker): The ESX code is packaged as [a vib or an offline depot](http://pubs.vmware.com/vsphere-60/index.jsp#com.vmware.vsphere.install.doc/GUID-29491174-238E-4708-A78F-8FE95156D6A3.html#GUID-29491174-238E-4708-A78F-8FE95156D6A3)
```

Comment:
Since we are still using the abbr "VDVS" in other places, here we should also keep it consistent, i.e. "VDVS driver".

Reply:
Sure!